### PR TITLE
subgraph: Misc updates

### PR DESCRIPTION
**Pull request by yondonfu**
**What does this pull request do? Explain your changes. (required)**

**Specific updates (required)**

- 
- 
- 

**How did you test each of these updates (required)**

**Does this pull request close any open issues?**

**Screenshots (optional):**

**Checklist:**

- [ ] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/subgraph/test/integration/test.js [hunk 1]
```diff
@@ -5,29 +5,30 @@ const RPC = require('../../utils/rpc')
...
 )
 const BondingManager = new web3.eth.Contract(
   BondingManagerABI,
   bondingManagerAddress,
+  defaults
 )
-const Token = new web3.eth.Contract(LivepeerTokenABI, livepeerTokenAddress)
-const PollCreator = new web3.eth.Contract(PollCreatorABI, pollCreatorAddress)
+const Token = new web3.eth.Contract(LivepeerTokenABI, livepeerTokenAddress, defaults)
+const PollCreator = new web3.eth.Contract(PollCreatorABI, pollCreatorAddress, defaults)
```
Comment (kyriediculous) [inline packages/subgraph/test/integration/test.js]: This is already sorted in https://github.com/livepeer/livepeerjs/pull/611/commits/e5952f2f24b5dbc97a862058bf72837fc2ab5205 I believe?

File: packages/subgraph/test/integration/test.js [hunk 2]
```diff
@@ -560,7 +554,7 @@ contract('Subgraph Integration Tests', accounts => {
     const pollAddress = subgraphPollData.data.polls[0].id
     const Poll = new web3.eth.Contract(PollABI, pollAddress)
 
-    Poll.methods.vote(1).send({ from: delegator5 })
+    await Poll.methods.vote(1).send({ from: delegator5 })
```
Comment (kyriediculous) [inline packages/subgraph/test/integration/test.js:568]: good catch, there's no guarantee this promise resolves before we sync the subgraph. Ganache's instant mining prevents the test from failing;

File: packages/subgraph/test/integration/test.js (hunk 1 above)
Comment (yondonfu) [reply packages/subgraph/test/integration/test.js]: Yep! Removed this.

**Review by adamsoffer — CHANGES_REQUESTED** [review]
Looks great thanks! Just noticed one other `await` I missed on L568.

**Comment by yondonfu** [comment]
> Just noticed one other await I missed on L568.

Fixed in 095a7f1

**Review by adamsoffer — APPROVED** [review]

**Pull request merged**